### ISSUE TITLE
spring-modulith-runtime causes log file is not created

### ISSUE THREAD

**Issue by razubuddy** [spring-projects-experimental/spring-modulith#169]
Steps to reproduce:
1. create bare spring boot application with spring initlaizr
2. add property `logging.file.name` to `application.properties`
```
logging.file.name=demo.log
```
3. add `spring-modulith-runtime` dependency in `build.gradle`
```
	runtimeOnly('org.springframework.experimental:spring-modulith-runtime:0.5.0')

```
4. run application `./gradlew bootRun`

Expected: `demo.log` file should be created

**Comment by odrotbohm** [comment]
I can't really imagine what would cause this? Care to prepare a minimal reproducer?

**Comment by odrotbohm** [comment]
It looks like the `logback.xml` that was supposed to live in `src/test/java` was accidentally located in `src/main/java` and thus leaks into user projects. Fix coming.

**Comment by odrotbohm** [comment]
Fix available in 0.6 snapshots.
